src/gui/search/searchwidget.cpp

```diff
@@ -112,8 +114,8 @@ SearchWidget::SearchWidget(MainWindow *mainWindow)
     iconSize.setWidth(iconSize.width() + 16);
     m_ui->tabWidget->setIconSize(iconSize);
 #endif
-    connect(m_ui->tabWidget, &QTabWidget::tabCloseRequested, this, &SearchWidget::closeTab);
-    connect(m_ui->tabWidget, &QTabWidget::currentChanged, this, &SearchWidget::tabChanged);
+    connect(m_ui->tabWidget->tabBar(), &MiddleClickTabBar::tabCloseRequested, this, &SearchWidget::closeTab);
```

Comment:
With https://github.com/qbittorrent/qBittorrent/pull/13465/files#r499801606 in effect, this should be reverted (the change below too).